### ISSUE TITLE
Support multiple failover unicast addresses

### ISSUE THREAD

**Issue by mhermsdorfer** [F5Networks/f5-declarative-onboarding#152]
### Is your feature request related to a problem? Please describe.
Currently, DO appears to only support a single FailoverUnicast address.  The schema validator allows you to define multiple FailoverUnicast classes, but DO appears to ignore all but the last defined FailoverUnicast class.  It is highly recommended, and required for stable HA operations to configure a minimum of one self-ip based failover heartbeat unicast address AND the management-ip for failover heartbeat unicast.  Currently it doesn't appear possible to follow this recommendation with DO.

From F5 docs on setting up HA:
K14135: Defining network resources for BIG-IP HA features: https://support.f5.com/csp/article/K14135

Network failover recommendations

To ensure the most stable network failover configuration, choose two channels for network failover communication. F5 considers it best practice to select both the management address and a TMM network address. This configuration avoids unexpected failover conditions, for example, while the BIG-IP system initializes and before the TMM network connectivity channels are fully established. To do this, you must configure a management and TMM failover unicast address.

K37361453: Configuring network failover for redundant VIPRION systems: https://support.f5.com/csp/article/K37361453

To ensure the most stable network failover configuration for redundant VIPRION systems, configure two channels for network failover communication. F5 considers it best practice to define a unicast and a multicast failover address for each VIPRION system in the device group. The unicast failover configuration uses a self IP address and Traffic Management Microkernel (TMM) switch port to communicate failover packets between each VIPRION system, and the multicast failover entry uses the management port to communicate failover packets between each VIPRION system. As an alternative to configuring the multicast failover option, you can define a unicast mesh using the management port for each VIPRION system. This is listed as an alternative option element.

### Describe the solution you'd like
The ability to define multiple failover unicast addresses.

For example, In DO declaration json:
```
        "failoverUnicastMgmt": {
            "class": "FailoverUnicast",
            "address": "<management-ip-addr> or string  "mangement-ip""
        },
        "failoverUnicastSelfSync": {
            "class": "FailoverUnicast",
            "address": "/Common/sync-self/address"
        }
        "failoverUnicastSelfVlanFoo": {
            "class": "FailoverUnicast",
            "address": "/Common/vlanFoo-self/address"
        }
```

Results in the following on the F5:
```
root@(bigip-vigo-01)(cfg-sync Standalone)(Active)(/Common)(tmos)# list cm device
cm device bigip-vigo-01.f5lab.net {
    ...
    unicast-address {
        {
            effective-ip 192.168.1.10
            effective-port 1026
            ip 192.168.1.10
        }
        {
            effective-ip 192.168.10.10
            effective-port 1026
            ip 192.168.10.10
        }
        {
            effective-ip management-ip
            effective-port 1026
            ip management-ip
        }
    }
}
```

### Describe alternatives you've considered
An alternative to uni-cast mesh is multicast for fail over heartbeats there's already an outstanding feature request for that: https://github.com/F5Networks/f5-declarative-onboarding/issues/128

### Additional context
Note, that currently DO lacks the ability to build a valid HA/failover configuration for Viprion as it lacks both: the ability to configure multiple unicast addresses, and the ability to configure a multicast failover heartbeat address.  Either a full unicast mesh or multicast heartbeat address is required to have a valid viprion configuration for failover.

Additionally, only having a single TMM based unicast address is not a recommended practice, since it will result in failover issues with missed heartbeats while waiting on TMM interfaces to come up.

**Comment by dstokesf5** [comment]
Thank you for the detailed feedback. I have added this request to our internal product backlog as AUTOTOOL-1675.

**Comment by dstokesf5** [comment]
This issue should be resolved in the 1.15.0 release.
